plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -90,6 +89,12 @@ public void testDropColumn()
         // Dropping columns are not supported by the connector
     }
 
+    @Override
+    public void testRenameTableAcrossSchema()
+    {
+        // Renaming tables across schemas are not supported by the connector
```

Comment:
nit: use `SkipExceptions` instead of commented out tests. That way we are aware that the test is either not implemented or the feature is not supported instead of getting a green test.

Reply:
done.